plugins/@grouparoo/intercom/src/lib/export-contacts/exportProfile.ts

```diff
@@ -300,7 +300,7 @@ async function deleteUser(
 }
 
 function formatVar(value) {
-  if (!value) {
```

Comment:
Yeah... these get me in trouble :/

what about `null`?

Reply:
Yeah, Salesforce is checking for both `null` and `undefined`

Reply:
since `value` is `null`, it will just be returned in line 310 (`return value = return null`). I did it explicitly for salesforce because it has some extra logic in there for getting custom default values.

Reply:
fair enough!